app/konnect/getting-started/spec/service.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Import an API Spec into Konnect
+no_version: true
+---
+
+Use the Service Hub to manage and share your APIs. Bring an API spec and publish
+it through the Service Hub to the Dev Portal.
+
+The Dev Portal is an API catalog that lets you document your Konnect Services
+and share them with your developers. Developers can use the Dev Portal to
+locate, access, consume, and register applications against the Services.
+
+This topic walks you through importing an API spec into a Konnect Service,
+and adding a description.
+
+## Prerequisites
+* OpenAPI spec in JSON or YAML format, or use the [sample vitalsSpec.yaml](/konnect/vitalsSpec.yaml)
+for testing.
+    _[Which versions of OpenAPI do we support?]_
+    _[Suggest using Insomnia to develop one, or is that too heavy-handed?]_
+* TBA: Required permissions
```

Comment:
For the purpose of the Getting Started guide, I suggest we use the Organization Admin team as the prerequisite. Otherwise, we'd have to do into details on roles and how that works.